azure_arc_servers_jumpstart/docs/onboard_server_win.md

```diff
@@ -1,23 +1,23 @@
 # Onboard an existing Windows server with Azure Arc
 
-The following README will guide you on how to connect an Windows machine to Azure Arc using a simple Powershell script.
```

Comment:
We are aligned with the official marketing name
https://docs.microsoft.com/en-us/powershell/scripting/overview?view=powershell-7.1